app/src/screens/Splash.tsx

```diff
@@ -90,6 +91,17 @@ const Splash: React.FC = () => {
     }
   }
 
+  const loadPersonNotificationDismissed = async (): Promise<void> => {
+    const dismissedData = await AsyncStorage.getItem('PersonCredentialOfferDismissed')
```

Comment:
Best to always wrap await in a try/catch unless it creates a unless try/catch. In this case I might consider putting the getItem and JSON.parse in a separate fn with a try/catch so it returns `undefined` if it throws.